addons/web/static/tests/views/list_tests.js

```diff
@@ -2930,11 +2965,116 @@ QUnit.module('Views', {
         assert.hasClass(list.$('tr.o_data_row:eq(0)'),'o_selected_row',
             "1st row should be selected");
         await testUtils.fields.triggerKeydown(list.$('tr.o_selected_row input[name="foo"]'), 'down');
-        assert.hasClass(list.$('tr.o_data_row:eq(0)'),'o_selected_row',
-            "1st row should still be selected");
+        assert.hasClass(list.$('tr.o_data_row:eq(1)'),'o_selected_row',
+            "2nd row should be selected");
+        list.destroy();
+    });
+
+    QUnit.test('navigation: moving down with keydown on last line (editable list not one2many)', async function (assert) {
+        assert.expect(0);
+
+        var list = await createView({
+            View: ListView,
+            model: 'foo',
+            data: this.data,
+            arch:   '<tree editable="bottom">'+
+                        '<field name="foo" required="True"/>'+
+                        '<field name="m2o" required="True"/>'+
+                        '<field name="m2m" required="True" widget="many2many_tags"/>'+
+                        '<field name="int_field"/>'+
+                    '</tree>',
+            //debug: true,
+        });
+
+        // await testUtils.dom.click(list.$('td:contains(-4)')); //int_field=-4 and foo=blip and m2o: 1 (Value 1)
```

Comment:
don't forget to remove commented out code

Reply:
yes, thanks, I still have to adapt that one...